<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-I87U9OT" failures="1" tests="4" name="Test" time="3.108" errors="0" timestamp="2024-01-03T02:23:52 IST">
  <testcase classname="Amezon_LoginPage.Loginpage_TestNG" name="dispaly" time="3.024">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method Amezon_LoginPage.Loginpage_TestNG.dispaly() didn&amp;apos;t finish within the time-out 3000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method Amezon_LoginPage.Loginpage_TestNG.dispaly() didn't finish within the time-out 3000
at java.base@18.0.2.1/java.lang.Thread.sleep(Native Method)
at app//Amezon_LoginPage.Loginpage_TestNG.dispaly(Loginpage_TestNG.java:34)
at java.base@18.0.2.1/java.lang.invoke.LambdaForm$DMH/0x0000000800190400.invokeVirtual(LambdaForm$DMH)
at java.base@18.0.2.1/java.lang.invoke.LambdaForm$MH/0x0000000800190c00.invoke(LambdaForm$MH)
at java.base@18.0.2.1/java.lang.invoke.Invokers$Holder.invokeExact_MT(Invokers$Holder)
at java.base@18.0.2.1/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base@18.0.2.1/java.lang.Thread.run(Thread.java:833)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- dispaly -->
  <testcase classname="Amezon_LoginPage.Loginpage_TestNG" name="show" time="0.001"/>
  <testcase classname="Amezon_LoginPage.Loginpage_TestNG" name="show" time="0.001"/>
  <testcase classname="Amezon_LoginPage.Loginpage_TestNG" name="show" time="0.001"/>
  <testcase classname="Amezon_LoginPage.Loginpage_TestNG" name="Test12" time="0.0">
    <ignored/>
  </testcase> <!-- Test12 -->
</testsuite> <!-- Test -->
